### DISCUSSION TITLE
RuntimeError: Job 69be6321-337e-4f5e-8a5e-04f8416aac72 failed for more than 3 times

### DISCUSSION THREAD

**Discussion by Shangguanying992** [deepmodeling/deepks-kit#65] [General]
When running my DeePKS model, I encountered the following error. 
The DeePKS water_single example can run correctly on my machine.
![屏幕截图 2023-12-16 134811](https://github.com/deepmodeling/deepks-kit/assets/147317022/5d9d1450-cd59-4630-9e9c-bcbfa556a054)

**Comment by y1xiaoc** [comment]
This error just means one of the deepks steps failed to finish. You may need to dig into the folders to find the real error message. I will convert this to a discussion as it might not be a bug since water_single example runs.
